lib/sauce.js

```diff
@@ -7,9 +7,11 @@ function Sauce() {
   var _tunnel;
 
   function stop(done) {
-    console.log('Closing Sauce Connect.');
+    console.log("Sauce Connect closing tunnel.");
     if(_tunnel) {
       _tunnel.close(function() {
+        console.log("Sauce Connect closed.");
```

Comment:
Please follow the project style-guide and use single quotes.

Reply:
Sorry about that :) Fixed